test/cloud-slack-dev-e2e/cloud_slack_dev_e2e_test.go

```diff
@@ -88,6 +88,8 @@ func TestCloudSlackE2E(t *testing.T) {
 	err := envconfig.Init(&cfg)
 	require.NoError(t, err)
 
+	cfg.Slack.Tester.CloudBasedTestEnabled = false // override property used only in the Cloud Slack E2E tests
```

Comment:
it would be good to differentiate a bit more it as we now have two things named cloud slack e2e tests, maybe sth like:
- cloud Slack enablement tests
- cloud Slack conversation tests

but let's follow up on this in a separate PR once you will be back @pkosiec